solarforecastarbiter/validation/tasks.py

```diff
@@ -408,6 +409,37 @@ def validate_daily_ac_power(observation, values):
             interpolation_flag, clipping_flag)
 
 
+def validate_daily_defaults(observation, values):
+    """
+    Run default daily validation checks on an observation.
+    Applies the validation for the observation's variable and then
+    the stale and interpolated validation. If the variable does
+    not have a defined validation function :py:func:`validate_defaults`
+    is used.
```

Comment:
Reverse sentence order (validate_defaults is used if...) or add comma after function.